scripts/python/docs-automation/sync_tocs.py

```diff
@@ -31,19 +31,20 @@ def filter_children(targeted_ns_list, known_namespaces):
 
     return amended_list
 
-
-def amend_href(input_string, repo_location, readme_suffix):
+def amend_href(toc_dict, repo_location, readme_suffix):
     suffix = "-" + readme_suffix + ".md"
+    input_string = toc_dict["href"]
 
     resolvable_path = os.path.join(repo_location, input_string.replace("~/", ""))
-    possible_target_readme = os.path.splitext(resolvable_path)[0] + "-" + suffix
+    possible_target_readme = os.path.splitext(resolvable_path)[0] + suffix
 
     if os.path.exists(possible_target_readme):
+        toc_dict["href"] = input_string.replace(".md", suffix)
+    else:
+        toc_dict.pop("href")
```

Comment:
I don't have enough context here but if you remove the href does this mean that there will not be a readme link provided in the ToC?

Reply:
Yes. This is intentional. I'd rather show NO readme than the wrong readme.

Reply:
Not only this, but if the same readme is requested from two different ToCs (latest and preview), the entire node ceases to render.